src/clusterfuzz/_internal/cron/project_setup.py

```diff
@@ -751,8 +752,14 @@ def _get_base_project_name(self, project):
         return base_project
     return project
 
-  def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
-                logs_bucket_name, backup_bucket_name):
+  def _sync_job(self,
+                project,
+                info,
+                corpus_bucket_name,
+                quarantine_bucket_name,
+                logs_bucket_name,
+                backup_bucket_name,
+                oss_fuzz_gb=None):
```

Comment:
what's passing this  in? do we need this as an argument?

Reply:
Ah I messed up a cherry pick. Let me fix this.